### ISSUE TITLE
Black screen due to Kopernicus failure to load body Moon 

### ISSUE THREAD

**Issue by KaioKei** [ballisticfox/RSS-Reborn#2]
Hello authors !
First, thank you for your amazing work.

Second, I followed your guide from the github page here : https://github.com/RSS-Reborn/RSS-Reborn/wiki/Installation.
Note that I installed RP-1 from CKAN using the express installation.

But after I freshly installed RSS Reborn (with Blackrack's volumetric clouds), I encounter an issue involving Kopernicus :
1. A Kopernicus warning message during the first startup of KSP after a fresh install of RSS Reborn (Manual) :
![bug_kopernicus_rss_reborn](https://github.com/user-attachments/assets/a931d097-fa26-47f7-96b9-da4706bde66a)
2. A black screen (infinite loading) after a second start of KSP (similar to https://github.com/RSS-Reborn/RSS-Reborn/issues/1, however the former user hasn't shared its solution)

There is indeed an error is KSP.log :

```
[EXC 17:20:58.065] NullReferenceException: Object reference not set to an instance of an object
	Kopernicus.Components.ModularScatter.PQSLandControl_LandClassScatter_AddScatterMeshController.Prefix (PQSLandControl+LandClassScatter __instance, System.Double& density) (at <0708e1bc41b144fe8a3d33f3f39c4d0b>:0)
	(wrapper dynamic-method) PQSLandControl+LandClassScatter.PQSLandControl+LandClassScatter.AddScatterMeshController_Patch1(PQSLandControl/LandClassScatter,PQ,double)
	PQSLandControl.OnQuadBuilt (PQ quad) (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQS.Mod_OnQuadBuilt (PQ quad) (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQS.BuildQuad (PQ quad) (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.Build () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.SetVisible () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateVisibility () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQ.UpdateSubdivision () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PQS.UpdateQuadsInit () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	(wrapper dynamic-method) PQS.PQS.StartSphere_Patch0(PQS,bool)
	PQS.ForceStart () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PSystemSetup.ForceInitPQS (PQS pqsIn, UnityEngine.Transform target) (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	PSystemSetup+<SetupLaunchSites>d__38.MoveNext () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <12e76cd50cc64cf19e759e981cb725af>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
[ERR 17:20:58.080] PQS Kerbin: Restarted

[EXC 17:21:44.521] NullReferenceException: Object reference not set to an instance of an object
	Scatterer.AtmoPreprocessor.ReleaseTextures () (at <fdee3865bbd54560b3f1af491ef04bf6>:0)
	Scatterer.AtmoPreprocessor.OnDestroy () (at <fdee3865bbd54560b3f1af491ef04bf6>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
[EXC 17:21:44.536] NullReferenceException: Object reference not set to an instance of an object
	Kopernicus.Injector.OnDestroy () (at <0708e1bc41b144fe8a3d33f3f39c4d0b>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```

Looking inside Logs/Kopernicus, there is indeed an error involving the loading of the body Moon (see attached file) :

[Kopernicus.log](https://github.com/user-attachments/files/16325098/Kopernicus.log)

```
[LOG 17:20:54]: [Kopernicus]: Configuration.Loader: Loaded Body: Sun
[LOG 17:20:54]: [Kopernicus]: Configuration.Loader: Loaded Body: Mercury
[LOG 17:20:55]: [Kopernicus]: Configuration.Loader: Loaded Body: Venus
[LOG 17:20:55]: [Kopernicus]: Configuration.Loader: Failed to load Body: Moon: Exception has been thrown by the target of a
```

Finally, looking inside Logs/Kopernicus/Moon.Body.Log (see attached file) :

[Moon.Body.log](https://github.com/user-attachments/files/16325105/Moon.Body.log)

```
[LOG 17:20:55]: Exception Was Recorded: Exception has been thrown by the target of an invocation.
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Reflection.MonoProperty.SetValue (System.Object obj, System.Object value, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] index, System.Globalization.CultureInfo culture) [0x0005d] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Reflection.PropertyInfo.SetValue (System.Object obj, System.Object value, System.Object[] index) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectMemberFromConfigurationNode (Kopernicus.ConfigParser.Parser+ParsedMemberInfo parsedMember, System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x00434] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectFromConfigurationNode (System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x000bd] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectMemberFromConfigurationNode (Kopernicus.ConfigParser.Parser+ParsedMemberInfo parsedMember, System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x00328] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectFromConfigurationNode (System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x000bd] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectMemberFromConfigurationNode (Kopernicus.ConfigParser.Parser+ParsedMemberInfo parsedMember, System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x00328] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.ConfigParser.Parser.LoadObjectFromConfigurationNode (System.Object o, ConfigNode node, System.String configName, System.Boolean getChildren) [0x000bd] in <b81c7e09ce34479e9d3831f2ed50b754>:0 
  at Kopernicus.Configuration.Loader.Kopernicus.ConfigParser.Interfaces.IParserEventSubscriber.PostApply (ConfigNode node) [0x000eb] in <0708e1bc41b144fe8a3d33f3f39c4d0b>:0 
[LOG 17:20:55]: Inner Exception Was Recorded: Object reference not set to an instance of an object
  at Kopernicus.Configuration.MaterialLoader.ScaledPlanetSimple2Loader.set_MainTexXnSetter (Kopernicus.Configuration.Parsing.Texture2DParser value) [0x00014] in <0708e1bc41b144fe8a3d33f3f39c4d0b>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <9577ac7a62ef43179789031239ba8798>:0 
```

Could you help me on this one please ? 🙏

**Comment by ballisticfox** [comment]
This is typically an issue with a mismatch of textures and configs version, I'd suggest redownloading the moon textures and the latest RSS-Configs

**Comment by KaioKei** [comment]
Hello @ballisticfox, 

It was exactly the cause of the issue, thank you so much !
I was mixing some textures from the last release of this repo and some from the word document's links in the patreon (like the Moon textures).
After I downloaded again every sources from this repo, the problem was fixed.

Like you mention in the [installation guide](https://github.com/RSS-Reborn/RSS-Reborn/wiki/Installation#rss-reborn-install-instructions) : we should not mix the configs and the textures between this repo and the word document in the patreon.
